resources/css/InaSAFE.css

```diff
@@ -1,5 +1,5 @@
 body {
-  font-family: 'Ubuntu', Fallback, sans-serif;
+  font-family: 'Ubuntu', 'Lucida Grande', 'Segoe UI', sans-serif;
```

Comment:
Lucida Grande is fallback for MacOS.
Segoe UI is for Windows 8+.
sans-serif is the default ugly one.